act-rules-format.bs

```diff
@@ -245,196 +267,224 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
+  </blockquote>
+</div>
 
 
 Change Log {#change-log}
-========================
+------------------------
 
-It is important to keep track of changes to the ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All changes to an ACT Rule that can change the [=outcome=] of a evaluation MUST be recorded in a change log. The change log can either be part of the rule document itself or be referenced from it.
+It is important to keep track of changes to the ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All changes to an ACT Rule that can change the [=outcomes=] of an evaluation MUST be recorded in a change log. The change log can either be part of the rule document itself or be referenced from it.
 
 Each new release of an ACT Rule MUST be identifiable with either a date or a version number. Additionally, a reference to the previous version of that rule MUST be available. For extensive changes, a new rule SHOULD be created and the old rule SHOULD be deprecated.
 
 <div class=example>
-  **Example**: An example of when a new rule ought to be created is when a rule that tests for the use of a `blink` element changes to instead look for any animated style changes. This potentially adds several new failures that were previously out of scope. Using that same rule as an example, adding an exception to allow `blink` elements positioned off screen can be done by updating the existing rule.
+  **Example:** An example of when a new rule ought to be created is when a rule that tests for the use of a `blink` element changes to instead look for any animated style changes. This potentially adds several new failures that were previously out of scope. Using that same rule as an example, adding an exception to allow `blink` elements positioned off screen can be done by updating the existing rule.
 </div>
 
 
-Issues List (optional) {#issues-list}
-=====================================
+Glossary {#glossary}
+--------------------
 
-An ACT Rule MAY include a list or a reference to a list of any known issues. The issues list would be used to record cases where the [=outcome=] of an ACT Rule was `failed` where it ought to have been `passed` or `inapplicable`, or vice versa. There are several reasons why this might occur, including:
+ACT Rules MUST have a glossary section. The glossary MUST contain the [=outcome=] definition, as well as any definitions used in [applicability](#applicability) and [expectations](#expectations) sections in the rule. Since changes to the definition change the rule, those definitions cannot be maintained independently of the rule. If a shared glossary is used for rules, any definition changes MUST be included in the [change log](#change-log) of all rules that use that definition.
```

Comment:
Maybe I missed when we discussed this in a call, but I'm not seeing why the glossary is a MUST, and why it needs to contain the _outcome_ definition (given that _outcome_ is defined in the spec already)…